configserver/pom.xml

```diff
@@ -236,6 +219,31 @@
       <version>${project.version}</version>
       <scope>compile</scope>
     </dependency>
+
+    <!-- Jersey, needed by serviceview  -->
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+      <scope>provided</scope>  <!-- TODO: Vespa 8: Set to compile if we get rid of the javax.ws.rs-api bundle -->
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-client</artifactId>
+    </dependency>
```

Comment:
Remove <scope>compile</scope> above, or add <scope>compile</scope> to all of these implicitly compile-scoped dependencies?